server/views/contact/ro.pug

```diff
@@ -0,0 +1,33 @@
+extends ../layout
+
+block content
+
+    div.pure-g
+        div.pure-u-1
+            include ../includes/back
+
+    div.pure-g.largePaddingBottom
+        div.pure-u-1.pure-u-2-3
```

Comment:
This overwrites 100% width column with 66% . Think you need div.pure-u-1.pure-u-md-2-3